### PR TITLE
Guardian newspaper page dated and other tweaks

### PR DESCRIPTION
**Pull request by jennysivapalan**
Following on from https://github.com/guardian/frontend/pull/11094 
- Adds newspaper pages for dates in the past e.g. `/theguardian/2015/nov/12`
- If the date is in the future then the page is redirected to `/theguardian`
- If the date is a Sunday it will show the Guardian for the Saturday edition. This is just temporary, when the `/theobserver' pages are created then this will all redirect as per rules on R2.

Other tweaks requested by Katie
- lowercase container names (except UK news and US news)
- Add a container for content with 5 items
- Adds the section navigation

screen shot 2015-11-13 at 12 18 00

### CODE REVIEW

File: applications/app/services/NewspaperQuery.scala [hunk 1]
```diff
@@ -18,32 +18,50 @@ case class ContentByPage(page: Int, content: ApiContent)
...
+    bookSectionContainers(DateTime.now(DateTimeZone.UTC))
+
+
+  def fetchPaperForDate(day: String, month: String, year: String): Future[List[FaciaContainer]] = {
+    val dateFormatUTC = DateTimeFormat.forPattern("yyyy/MMM/dd").withZone(DateTimeZone.UTC)
+
+    val date = dateFormatUTC
+      .parseDateTime(s"$year/$month/$day")
+      .withTimeAtStartOfDay()
+      .plusDays(1)
```
Comment (JustinPinner) [inline applications/app/services/NewspaperQuery.scala]: Watch this bit. We're setting the date to today at 23:59:59 which is fine if it's used as a to-date, but maybe not as a from-date.

File: applications/app/services/NewspaperQuery.scala [hunk 2]
```diff
@@ -108,4 +125,6 @@ object TodaysNewspaperQuery extends ExecutionContexts with Dates with Logging {
   }
 
   private def getNewspaperPageNumber(content: ApiContent) = content.fields.getOrElse(Map.empty).get("newspaperPageNumber").map(_.toInt)
+
+  def lowercaseDisplayName(s: String) = if(s.equals("UK news") || s.equals("US news")) s else s.toLowerCase()
```
Comment (JustinPinner) [inline applications/app/services/NewspaperQuery.scala]: :smile:

**Comment by JustinPinner** [comment]
Looks good. If you're sure that date.plusDays(1).minusSeconds(1) won't trip up the behaviour then :+1:

**Comment by jennysivapalan** [comment]
Okay thanks, I'll check that.

File: applications/app/services/NewspaperQuery.scala (hunk 1 above)
Comment (jennysivapalan) [inline applications/app/services/NewspaperQuery.scala]: Turns out I don't need this so will remove it.

**Pull request merged**